### PR TITLE
plugins: slack: update branch manager whitelist for v1.15

### PR DESCRIPTION
**Pull request by idealhack**
/sig contributor-experience
/sig release

xref: https://github.com/kubernetes/test-infra/pull/11105

### CODE REVIEW

File: prow/plugins.yaml [hunk 1]
```diff
@@ -206,7 +202,10 @@ slack:
         release-1.14:
             - aleksandra-malinowska # 1.14 patch release team
             - tpepper # 1.14 patch release team
-            - hoegaarden # 1.14 branch manager
+            - hoegaarden # 1.14 patch release team
+        release-1.15:
+            - bubblemelon # 1.15 branch manager
```
Comment (idealhack) [inline prow/plugins.yaml]: I'm not sure if about this since @Bubblemelon has a different username on slack

Comment (cblecker) [reply prow/plugins.yaml]: These are github usernames, not slack usernames.

Comment (idealhack) [reply prow/plugins.yaml]: Thanks! So the mention won't work?

File: prow/plugins.yaml [hunk 2]
```diff
@@ -206,7 +202,10 @@ slack:
         release-1.14:
             - aleksandra-malinowska # 1.14 patch release team
             - tpepper # 1.14 patch release team
-            - hoegaarden # 1.14 branch manager
+            - hoegaarden # 1.14 patch release team
+        release-1.15:
```
Comment (fejta) [inline prow/plugins.yaml]: https://github.com/kubernetes/sig-release/blob/master/releases/release-1.15/release_team.md

Comment (idealhack) [reply prow/plugins.yaml]: thanks! we are not sure about other shadows yet, I will follow up with this :)

**Pull request merged**